app/workers/step_by_step_draft_update_worker.rb

```diff
@@ -24,8 +24,18 @@ def update_draft
 
   def update_assigned_to
     unless step_by_step_page.has_draft?
+      generate_internal_change_note
       step_by_step_page.assigned_to = @current_user
       step_by_step_page.save
     end
   end
+
+  def generate_internal_change_note
+    change_note = step_by_step_page.internal_change_notes.new(
+      author: @current_user,
+      description: "Draft created by #{@current_user}",
+      created_at: Time.now
```

Comment:
You shouldn't need to pass in the `created_at` time, as this field should be auto-populated by ActiveRecord with the current date and time.

The reason that it is required is, we've added a validation to the `InternalChangeNote` model that forces the `created_at` date to be passed in. If you remove that validation, your tests will pass, and the created_at date will still (always) be updated.

See line: https://github.com/alphagov/collections-publisher/blob/master/app/models/internal_change_note.rb#L2